assignment3/graph.dart

```diff
@@ -0,0 +1,171 @@
+import 'dart:io';
+import 'util.dart';
+import 'functions.dart';
+
+class Graph {
```

Comment:
It's not a graph, the class you've created is of a `Node`. Rename it

Reply:
renamed it to Node